docs/aerodromes/Woomera.md

```diff
@@ -0,0 +1,44 @@
+---
+  title: Woomera (YPWR)
+---
+
+--8<-- "includes/abbreviations.md"
+
+## Positions
+
+| Name               | Callsign       | Frequency        | Login Identifier              |
+| ------------------ | -------------- | ---------------- | --------------------------------------|
+| Woomera ADC    | Woomera Tower  | 118.300         | WR_TWR        |
+| Woomera ATIS    |   | 118.100         | YPWR_ATIS       |
+
+## Airspace
+
+By default, WR ADC owns the **R222F** Restricted Area. This airspace has vertical limits of `SFC` to `F125`. When WR ADC is active, this airspace is classfied as Class D.
+
+Refer to [Class D Tower Skills](../../controller-skills/classdtwr) for more information.
+
+## Surveillance
+
+WR TWR is permitted to use Surveillance standards for separation. Surveillance coverage can be expected to be available at all levels within R222F.  
+For simulation purposes, visual separation is assumed to exist below the cloud base, and within 5nm. Visual separation can still be used to separate from aircraft on an instrument approach, below the cloud base.
+
+## Coordination
+### WR ADC / ASP(WRA)
+
+A 'next' call is made for all aircraft when they are next to depart. WR ADC must inform ASP(WRA) if the aircraft does not depart within **2 minutes** of the next call.
```

Comment:
Example please. And a next call shall only be made for aircraft departing in to ASP(WRA) CTA.